MonoGame.Framework/Graphics/Texture2D.DirectX.cs

```diff
@@ -268,9 +269,8 @@ private void PlatformSaveAsPng(Stream stream, int width, int height)
 #if WINDOWS_STOREAPP
             SaveAsImage(BitmapEncoder.PngEncoderId, stream, width, height);
```

Comment:
If it works on one DX platform I suspect it should work on all.  So you can remove the `#if WINDOWS_STOREAPP` here.